.github/workflows/ci.yml

```diff
@@ -71,6 +71,11 @@ jobs:
       run: |
         pytest --random-order --cov=freqtrade --cov=freqtrade_client --cov-config=.coveragerc
 
+    - name: Run Pester tests with code coverage
+      shell: pwsh
+      run: |
+        Invoke-Pester -Path tests -EnableExit
```

Comment:
Aside from the wrong "run" location - i think it's also implemented wrongly

Running this in (windows CI) yields the following error:

```
WARNING: You are using Legacy parameter set that adapts Pester 5 syntax to Pester 4 syntax. This parameter set is deprecated, and does not 
work 100%. The -Strict and -PesterOption parameters are ignored, and providing advanced configuration to -Path (-Script), and -CodeCoverage 
via a hash table does not work. Please refer to https://github.com/pester/Pester/releases/tag/5.0.1#legacy-parameter-set for more
information.
```